web/templates/buddy_list/view_all_subscribers.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="buddy-list-user-link view-all-subscribers-link">
-    <a class="right-sidebar-wrappable-text-container" href="{{stream_edit_hash}}">
+<div class="buddy-list-user-link view-all-subscribers-link hide">
+    <a class="right-sidebar-wrappable-text-container">
```

Comment:
I wonder if it's best practice to still have an `href`, even it's empty? I'll leave this as a question for Tim when he reviews it, or you could try to find the answer in documentation somewhere.